File: CS/JQueryValidation/Controllers/HomeController.cs
using System;
using System.Web;
using System.Web.Mvc;
using JQueryValidation.Models;
using DevExpress.Web.Mvc;

namespace JQueryValidation.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            return JQueryValidation();
        }
        [AcceptVerbs(HttpVerbs.Get)]
        public ActionResult JQueryValidation() {
            return View("JQueryValidation", new ValidationData());
        }
        [AcceptVerbs(HttpVerbs.Post)]
        public ActionResult JQueryValidation([ModelBinder(typeof(DevExpressEditorsBinder))] ValidationData validationData) {
            if(ModelState.IsValid)
                return View("JQueryValidationSuccess");
            return View("JQueryValidation", validationData);
        }
        public ActionResult JQueryValidationSuccess() {
            return View("JQueryValidationSuccess");
        }
    }
}

File: VB/JQueryValidation/Controllers/HomeController.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Web
Imports System.Web.Mvc
Imports JQueryValidation.Models
Imports DevExpress.Web.Mvc

Namespace JQueryValidation.Controllers
	Public Class HomeController
		Inherits Controller
		Public Function Index() As ActionResult
			Return JQueryValidation()
		End Function
		<AcceptVerbs(HttpVerbs.Get)> _
		Public Function JQueryValidation() As ActionResult
			Return View("JQueryValidation", New ValidationData())
		End Function
		<AcceptVerbs(HttpVerbs.Post)> _
		Public Function JQueryValidation(<ModelBinder(GetType(DevExpressEditorsBinder))> ByVal validationData As ValidationData) As ActionResult
			If ModelState.IsValid Then
				Return View("JQueryValidationSuccess")
			End If
			Return View("JQueryValidation", validationData)
		End Function
		Public Function JQueryValidationSuccess() As ActionResult
			Return View("JQueryValidationSuccess")
		End Function
	End Class
End Namespace
